.editorconfig

```diff
@@ -9,5 +9,8 @@ charset = utf-8
 trim_trailing_whitespace = true
 insert_final_newline = true
 
+[*.hbs]
+indent_style = space
```

Comment:
For what is this used?

Reply:
This just reflects what the code has. The handlebar-files is saw were all indented with spaces, or mixed tabs and spaces.

If the indentation should be tabs, then all files should be converted. Since they were not, I assumed nobody else cared.

Reply:
ok thanks